src/MetadataUtility/Models/Microphone.cs

```diff
@@ -4,20 +4,12 @@
 
 namespace MetadataUtility.Models
 {
-    using Newtonsoft.Json;
-
     /// <summary>
     /// Holds information regarding a microphone
     /// that is attached to the device.
     /// </summary>
     public record Microphone
     {
-        /// <summary>
-        /// Gets microphone number according to the sensor (1, 2, etc.).
-        /// Used internally to correlate microphone metadata correctly.
-        /// </summary>
-        public int Channel { get; init; }
-
```

Comment:
Channel should be present, why did you remove it?

Reply:
I originally used this to keep track of which microphone data belong to which microphone, pulling the channel number from the microphone vorbis comments (ex. MicrophoneTypeX where X is channel). I wasn't sure if this actually was the channel number though, so I removed it. Looking at the log files, they're referred to as microphone channel A and channel B. Could bring them back using the values from the vorbis comments (as integers?), or convert to alphabetical?

Reply:
Because this structure is essentially removing the order we should preserver the order in which the information was conveyed to us.

Channel should be a index (A = 0, and so on)
Maybe add a ChannelName property as well so we don't lose the `A`/`B` part of the information.